packages/web-app-files/src/views/Favorites.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div class="oc-flex">
     <files-view-wrapper>
-      <app-bar :side-bar-open="sideBarOpen" />
+      <app-bar :display-view-mode-switch="true" :side-bar-open="sideBarOpen" />
```

Comment:
Side note: This already is an example where we don't want the `condensed` view mode (or at least it doesn't make too much sense, since it is not different from the default list view mode), but will want to have the `tiles` view mode available in the future. :-/ Let's keep it like you have it here for the time being.

Reply:
_soon_